src/baseconverter.js

```diff
@@ -1,18 +1,30 @@
 'use strict';
 
+
 function normalize(digitArray) {
-  while (digitArray[0] === 0) {
-    digitArray.shift();
+  const length = digitArray.length;
```

Comment:
I don't think this is actually used any more by anything, as we now (indirectly) use this: https://github.com/cryptocoinjs/base-x